Installer/JobsSetup/Product.wxs

```diff
@@ -72,6 +72,16 @@
 	                    Name="Version"
 	                    Type="raw" />
 	  </Property>
+
+	  <!-- powerJobs -->
+	  <Property Id="POWERJOBSINSTALLEDVERSION">
+		<RegistrySearch Id="PowerGateInstalledVersion"
```

Comment:
Azure DevOps is failing:

`Installer\JobsSetup\Product.wxs(69,0): Error LGHT0091: Duplicate symbol 'RegLocator:PowerGateInstalledVersion' found. This typically means that an Id is duplicated. Check to make sure all your identifiers of a given type (File, Component, Feature) are unique.`

Reply:
fixed